tools/publish.sh

```diff
@@ -16,6 +16,7 @@ crates=(
     bevy_audio
     bevy_core
     bevy_diagnostic
+    bevy_hierarchy
```

Comment:
This is interesting; I wonder if this would publish a not-very-useful standalone `bevy_hierarchy` 0.5...

Reply:
I don't know how this works; I was just following the advice in `CONTRIBUTING.md`. That advice told me to add it, but I don't know if I did something wrong.